snooty/parser.py

```diff
@@ -1046,11 +1047,23 @@ def _locate_text(text: str) -> int:
             image_argument = options.get("icon")
             url_argument = options.get("url")
 
-            if image_argument:
-                self.validate_and_add_asset(doc, image_argument, line)
             if url_argument and not url_argument.startswith("http"):
                 self.validate_relative_url(url_argument, line)
 
+            # for cards - if the image is a path, we need a dark mode version as well
+            if key == "mongodb:card":
+                if image_argument and "/" in image_argument:
+                    self.diagnostics.append(CardIconString(image_argument, line))
```

Comment:
similar to @rayangler s comment above but wouldn't [this PR](https://github.com/mongodb/docs-ecosystem/pull/972/files) still have a warning ? 

think we should allow name and path, but require a `icon-dark` if its a path